Tools/jit/template.c

```diff
@@ -79,6 +91,7 @@ _JIT_ENTRY(_PyInterpreterFrame *frame, PyObject **stack_pointer, PyThreadState *
     PATCH_VALUE(uint16_t, _oparg, _JIT_OPARG)
     PATCH_VALUE(uint64_t, _operand, _JIT_OPERAND)
     PATCH_VALUE(uint32_t, _target, _JIT_TARGET)
+    PATCH_VALUE(uint16_t, _exit_index, _JIT_EXIT_INDEX)
     // The actual instruction definitions (only one will be used):
     if (opcode == _JUMP_TO_TOP) {
```

Comment:
Good idea, but for another PR.
It will help with any loop invariant code motion we want to do in the optimizer as well.
https://github.com/python/cpython/issues/117062